dpnp/dpnp_iface_histograms.py

```diff
@@ -752,3 +775,310 @@ def histogram_bin_edges(a, bins=10, range=None, weights=None):
     a, weights, usm_type = _ravel_check_a_and_weights(a, weights)
     bin_edges, _ = _get_bin_edges(a, bins, range, usm_type)
     return bin_edges
+
+
+def _histdd_validate_bins(bins):
+    for i, b in enumerate(bins):
+        if numpy.ndim(b) == 0:
+            if b < 1:
+                raise ValueError(
+                    f"'bins[{i}' must be positive, when an integer"
+                )
+        elif numpy.ndim(b) == 1:
+            # will check for monotonicity later
+            pass
+        else:
+            raise ValueError(
+                f"'bins[{i}]' must be either scalar or 1d array-like,"
+                + f" but it is {type(b)}"
+            )
+
+
+def _histdd_check_monotonicity(bins):
+    for i, b in enumerate(bins):
+        if dpnp.any(b[:-1] > b[1:]):
+            raise ValueError(
```

Comment:
Could you please add a test to cover that?

Reply:
We already have test for monotocity:
https://github.com/IntelPython/dpnp/pull/2143/files#diff-1627b9bd31e3c8c8dd1e2ce36d69e6544a2d4d2ef9689d9d22c815462730c355R730

And another one in cupy tests

Reply:
Removed this check as we fails earlier inside `_get_bin_edges` which also contains monotonicity check